README.md

```diff
@@ -1,158 +1,270 @@
+<!DOCTYPE html>
+<html>
+<body>
+  <!-- HEADER/NAME -->
+  <h1 id="name"><span style="font-variant: small-caps;">Ewon Ignition Demo</span></h1>
+
+  <!-- TABLE OF CONTENTS -->
+  <h2 id="toc"><span style="font-variant: small-caps;">Table of Contents</span></h2>
+
+  <ol>
+    <li><a href="#information">Information</a>
+      <ol>
+        <li><a href="#about">About</a></li>
+        <li><a href="#system-design">System Design</a></li>
+        <li><a href="#trademark-information">Trademark Information</a></li>
+      </ol>
+    </li>
+    <li><a href="#getting-started">Getting Started</a>
+      <ol>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+      </ol>
+    </li>
+    <li><a href="#installation">Installation</a>
+      <ol>
+        <li><a href="#ewon-setup">Ewon Setup</a></li>
+        <li><a href="#ignition-setup">Ignition Setup</a></li>
+      </ol>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#troubleshooting">Troubleshooting</a>
+      <ol>
+        <li><a href="#cannot-log-in-trial-expired">Cannot log in, trial expired</a></li>
+        <li><a href="#error-writing-to-tag-bad_notfound">Error writing to tag, Bad_NotFound<a/></li>
+      </ol>
+    </li>
+    <li><a href="#appendices">Appendices</a>
+      <ol>
+        <li><a href="#setting-up-an-ewon">Setting Up an Ewon</a></li>
+        <li><a href="#obtaining-a-talk2m-developer-id">Obtaining a Talk2M Developer ID</a></li>
+        <li><a href="#accessing-an-ewon-devices-webpage">Accessing an Ewon Device's Webpage</a></li>
+        <li><a href="#installing-and-configuring-ignition">Installing and Configuring Ignition</a></li>
+        <li><a href="#restarting-ignition">Restarting Ignition</a></li>
+        <li><a href="#obtaining-an-ewons-name">Obtaining an Ewon's Name</a></li>
+      </ol>
+    </li>
+  </ol>
+
+  <!-- INFORMATION -->
+  <h2 id="information"><span style="font-variant: small-caps;">Information</span></h2>
+
+  <!-- INFORMATION/ABOUT -->
+  <h3 id="about"><span style="font-variant: small-caps;">About</span></h3>
+  <p>Ewon Ignition Demo is a project that showcases the features of integrating the HMS Networks’ Ewon platform and the Inductive Automation Ignition platform.</p>
+  <p>Ewon Ignition Demo was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please visit the Ewon Developer website at <a href="https://developer.ewon.biz/" class="uri">https://developer.ewon.biz/</a> or contact us at <a href="mailto:us-services@hms.se">us-services@hms.se</a>.</p>
```

Comment:
This line needs "The" at the start.

Reply:
Ewon Ignition Demo is used several places in the readme as the first words in the sentence.  This reads a bit awkward to me.

Reply:
updated these lines